src/Examples/UnitModels/Operations/heat_exchanger_0D_testing.ipynb

```diff
@@ -98,11 +100,28 @@
     "m.fs.properties_tube = GenericParameterBlock(**configuration)"
```

Comment:
A minor comment, but I would be inclined to separate the imports from creating the flowsheet for clarity (and to allow a markdown box in between to make this clear). Note that the text above only mentioned importing components, not actually building the model.

Also, you appear to be importing the old activity coefficient package for BT systems, but are not actually using it - this should be removed (or if you need it later, I would import it then so that you can explain why).